solutions/usaco-1159.mdx

```diff
@@ -2,17 +2,102 @@
 id: usaco-1159
 source: USACO Silver 2021 December
 title: Connecting Two Barns
-author: Nathan Gong
+author: Nathan Gong, Jesse Choe
 ---
 
 [Official Analysis (C++)](http://usaco.org/current/data/sol_prob2_silver_dec21.html)
 
-## Implementation - DFS + Two Pointers
+## Implementation
```

Comment:
hm maybe we should specify the type of solution for each one because the java one uses a different approach

Reply:
Yeah, I changed it.